app/Providers/AppServiceProvider.php

```diff
@@ -115,6 +115,10 @@ public function boot(): void
                 NodeVersionsCheck::new(),
             ]);
         }
+
+        Gate::before(function (User $user, $ability) {
+            return $user->isRootAdmin() ? true : null;
+        });
```

Comment:
```suggestion
        Gate::before(fn (User $user) => $user->isRootAdmin() ? true : null);
```
Removed unused var and used arrow function

Reply:
That's a commit from main, not from this PR.